README.md

```diff
@@ -1,72 +1,60 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Original design idea by Cindy Shin in Behance, in association with the Creative Commons [licence](https://creativecommons.org/licenses/by-nc/4.0/) of the design.
 
-> Description the project.
 
+## Microverse HTML CSS Javascript-Capstone
+This Capstone Project marks the first milestone in HTML CSS and Javascript.
 
-## Built With
+Set up a repository and prepare it for development using correct git workflow. Build the three sections of the mobile version and use media queries to create the desktop version.
+
+## Screenshot
+![Screenshot (3)](https://user-images.githubusercontent.com/106156077/192014541-4d08a5a4-9785-44c7-ba23-fac0289ffef6.png)
 
-- Major languages
-- Frameworks
-- Technologies used
 
-## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+## Built With
 
+- HTML CSS & JAVASCRIPT
+- Git and Github
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
+- Copy link to this repository.
 
-### Run tests
+- Clone the ripository into local computer in git bash.
 
-### Deployment
+- Go to the repository folder using command prompt cd  capstone-project.
 
+- Install the developer dependencies for linters run npm install.
 
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+  Diana Duya
 
-👤 **Author2**
+- GitHub: [@Diana-Duya](https://github.com/Diana-Duya)
+- Twitter: [@Dianaduya2](https://twitter.com/Dianaduya2)
+- LinkedIn: [Diana Duya](https://www.linkedin.com/in/diana-duya-3088681a5)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Diana-Duya/Capstone-Project/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
```

Comment:
- [ ] Hi you are required to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in your Acknowledgments section, as required by the [Creative Commons License](https://creativecommons.org/licenses/).